src/main/java/TaskManager.java

```diff
@@ -0,0 +1,87 @@
+public class TaskManager {
+    private Task[] tasks = new Task[100];
+    private int taskCount = 0;
+    static Greet greet;
+
+    public TaskManager() {
+    }
+
+    public int getTaskCount() {
+        return taskCount;
+    }
+
+    public void markTask(int number){
+        greet.printDecoration();
+        if(number > taskCount || number < 1){
+            System.out.println("Invalid Number! Let's try again...");
+        }
+        else {
+            if(tasks[number - 1].isMarked())
+            {
+                System.out.println("Err, this task is already marked...");
+            }
+            else {
+                tasks[number - 1].setMarked(true);
+                System.out.println("Nice! I've marked this task as done: ");
+                System.out.println(tasks[number - 1]);
+            }
+        }
+        greet.printDecoration();
+    }
+
+    public void unmarkTask(int number){
+        greet.printDecoration();
+        if(number > taskCount || number < 1){
+            System.out.println("Invalid Number! Let's try again...");
+        }
+        else {
+            if(!tasks[number - 1].isMarked()){
+                System.out.println("I cannot unmark something that was never marked...");
+            }
+            else {
+                tasks[number - 1].setMarked(false);
+                System.out.println("Okay Boss! The following task has been unmarked: ");
+                System.out.println(tasks[number - 1]);
+            }
+        }
+        greet.printDecoration();
+    }
+
+    public void addToTasks(String taskName){
+        greet.printDecoration();
+        tasks[taskCount] = new Todo(taskName);
+        System.out.println("added: " + tasks[taskCount]);
+        taskCount++;
+        System.out.println("You now have " + taskCount + " tasks in the list.");
+        greet.printDecoration();
+    }
+
+    public void addToTasks(String type, String taskName,String date){
+        greet.printDecoration();
+        if(type == "E")
+        {
```

Comment:
Consider putting the opening brace on the same line as the if statement as per the K&R standards.